app/views/includes/commonLinkElements.scala.html

```diff
@@ -0,0 +1,23 @@
+@*
```

Comment:
Could name this one commonHeadElements, rather than LinkElements .. since it also contains some `<meta>` as well as links to styling - and it's destined for the `<head>` of the page.